README.md

```diff
@@ -1,31 +1,31 @@
 # JAVASCRIPT ADVANCED DISCUSSION GROUP
-Join us for a round table of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
+Join us for an hour of of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
 ## Pre-requisites:
-- JavaScript Track: JS130, Up to Testing
+- JavaScript Track: JS130
 - Ruby Track: JS225
-- Understand Bloom's [Taxonomy Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
+- Check out Bloom's Taxonomy with one of these: [Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
```

Comment:
How about: 'Check out Bloom's Taxonomy for inspiration with one of these:' or 'Check out Bloom's Taxonomy for learning more about how to deepen your knowledge of a material'

Reply:
Love it! Great idea @katarinarosiak !